test/format_coverage_test.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:test/test.dart';
+
+import '../bin/format_coverage.dart';
+
+void main() {
+  Directory testDir;
+  setUp(() {
+    testDir = Directory.systemTemp.createTempSync('coverage_test_temp');
```

Comment:
https://pub.dev/packages/test_descriptor does this dance really well – not sure if want to deal with that, though...

Reply:
Yeah I thought about that but given that it will only be used for a single test I decided against adding the dep.